Jenkinsfile

```diff
@@ -40,7 +42,7 @@ pipeline {
             echo 'DRY RUN: Skipping release tag creation'
           }
           else {
-            releaseTag.create(params.RELEASE_NAME)
+            env.GIT_TAG = releaseTag.create(params.RELEASE_NAME)
```

Comment:
I'm still a bit skeptical of passing state via environment variables. I raised this concern in prior PRs, so I apologize if I missed you response in one of the many PRs I already approved.